cnd/tests/ethereum_missing_blocks.rs

```diff
@@ -95,9 +107,11 @@ async fn find_transaction_in_missing_block_with_big_gap() {
         ],
         vec![(transaction.hash, receipt.clone())],
     );
-    let block1: Block<Transaction> = include_json_test_data!(
-        "./test_data/ethereum/find_transaction_in_missing_block/block1.json"
+    // Set start_of_swap to time of block 2, this allows us to go back to block 1.
```

Comment:
In general, in this file we are writing tests for missed blocks between calls to `latest_block()`, as opposed to old blocks that were missed because we started looking for a transaction too late, which go in `ethereum_old_blocks.rs`. 

I would be keen to rename this file to `ethereum_missed_blocks.rs` btw. What do you think?

Reply:
I like the rename idea, will look over both and make sure I've not muddied the water, so to speak.